### PR TITLE
welcome file

### PR DESCRIPTION
**Pull request by brntbeer**
this references #1 where we had a discussion about myself. I want this more persisted, so i added it here!

### CODE REVIEW

**Comment by brntbeer** [comment]
I think this looks pretty good. @jaw6 thoughts?

File: welcome.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Personal Information
+- Name: Brent Beer
+- Location: San Francisco, CA
+- Favorite Restaurant: Chez Maman
```
Comment (brntbeer) [inline welcome.md:4]: @moby Have you ever been here?

**Comment by brntbeer** [comment]
@PatrickBM can i get a :+1: ?

**Comment by patrickmckenna** [comment]
:+1:

**Pull request merged**